src/main/groovy/nebula/plugin/info/java/InfoJavaPlugin.groovy

```diff
@@ -78,6 +85,31 @@ class InfoJavaPlugin implements Plugin<Project>, InfoCollectorPlugin {
         }
     }
 
+    @CompileDynamic
+    private static JavaCompatibility findSourceAndTargetCompatibility(Project project) {
```

Comment:
There could be cases where `compileJava` and `compileGroovy` have different target/source jdks... This is something users shouldn't do honestly but Gradle allows you to, since compile tasks are independent. Maybe some good feedback for them,